client/android/build.gradle

```diff
@@ -2,16 +2,17 @@
 
 ext {
     // dependency versions
-    googlePlayServicesVersion = "11.0.4"
+    googlePlayServicesVersion = "12.0.1"
 }
 
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
-        classpath 'com.google.gms:google-services:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.google.gms:google-services:3.2.0'
```

Comment:
i thought we would be including firebase stuff here, no?

Reply:
// NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files

Reply:
These are build deps, not application deps... *i think*